client/state/sites/hooks/use-billing-history.ts

```diff
@@ -13,7 +13,7 @@ const fetchPastBillingTransactions = () =>
 		apiVersion: '1.3',
 	} );
 
-export const usePastBillingTransactions = () => {
+export const usePastBillingTransactions = ( isInSignup: boolean ) => {
```

Comment:
If the only purpose of `isInSignup` is to disable this hook, I think that a disabled parameter would be more semantic and reusable.

```suggestion
export const usePastBillingTransactions = ( disabled: boolean ) => {
```

Reply:
This is a great suggestion. Although the changes actually end up being unrelated to this PR, I'm making the changes since they're a small update 😃

_Edit:_ Resolved in [e8adbc6](https://github.com/Automattic/wp-calypso/pull/81816/commits/e8adbc618cbd4d717eff87fd5684fb6744047159)